CMakeLists.txt

```diff
@@ -26,13 +26,56 @@ if(POLICY CMP0071)
 endif()
 
 # Set up vcpkg
-if(DEFINED ENV{VCPKG_ROOT} AND NOT DEFINED VCPKG_ROOT)
-    set(VCPKG_ROOT "$ENV{VCPKG_ROOT}")
+if(WIN32 OR APPLE)
+  option(VCPKG "Use vcpkg for dependencies" ON)
+else()
+  option(VCPKG "Use vcpkg for dependencies" OFF)
 endif()
 
-if(DEFINED VCPKG_ROOT)
+if(VCPKG)
+  message(STATUS "Using dependencies from vcpkg")
+
   set(X_VCPKG_APPLOCAL_DEPS_INSTALL TRUE)
 
+  if(NOT DEFINED ENV{VCPKG_BINARY_SOURCES})
+    set(ENV{VCPKG_BINARY_SOURCES} "clear;nuget,mixxx,read;default;")
+  endif()
+
+  set(ENV{VCPKG_DISABLE_COMPILER_TRACKING} "1")
```

Comment:
```suggestion
  set(ENV{VCPKG_DISABLE_COMPILER_TRACKING} "true")
```

Reply:
Do we want that for the GitHub Workflow case? I don't think so

Reply:
This is required for the packages built on GitHub Actions to be usable locally.